docs/src/components/previews/CloudImagePreview.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+
+import { useState } from 'react';
+import { ProgressCircle } from '@voussoir/progress';
+import { getDefaultSrcSet } from '../../utils';
+import { CONTENT_MAX_WIDTH_DESKTOP } from '../../constants';
+
+export function CloudImagePreview(props: any) {
```

Comment:
I wonder how hard it would be to type this? Using something like `InferRenderersForComponentBlocks` but with preview props?

Reply:
Ahh yes, I will look into this.

Reply:
I wasn't able to do this because of the circular reference, so have just left it as `any`. I think this will be replaced fairly soon so I'm thinking it's not worth typing manually but I'm happy to if you'd prefer.

Reply:
Nah all good. Let's leave it